_i18n/de/resources/user-guides/tor_wallet.md

```diff
@@ -1,85 +1,85 @@
-{% include disclaimer.html translated="no" translationOutdated="no" %}
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
 
-Below we'll show an example configuration that allows you to run a Monero daemon (eg on a home server or VPS) that you can connect to from another computer running your wallet.  We do this over the Tor anonymity network to retrieve the transaction information needed by your wallet.  The benefit of this approach is that the daemon (`monerod`) can stay on all of the time sending / receiving blocks, while the wallet can connect when needed and have access to the full blockchain. [Monerujo](https://www.monerujo.io/) should also work via [Orbot](https://guardianproject.info/apps/org.torproject.android/).  Because Tor hidden services provide encryption and authentication, you can be confident that your RPC credentials will not be sent in the clear.  Tor also solves problems often seen on home servers related to port-forwarding, IP addresses changing, etc -- it just works.  This setup will also obfuscate the fact that you are connecting to a remote Monero node. Tested with Monero `v0.15.0.1` connecting a Mac laptop wallet to a remote Linux node (Ubuntu 18.04.2).
+Unten findest du eine beispielhafte Konfiguration, mit welcher es dir möglich ist, einen Monero-Hintergrunddienst (etwa auf einem Privatserver oder VPS) zu betreiben, zu dem du von einem anderen Rechner, auf welchem dein Wallet läuft, verbinden kannst. Über das Anonymitätsnetzwerk Tor werden die für dein Wallet benötigten Transaktionsinformationen wiederhergestellt. Das Gute an dieser Herangehensweise ist, dass der Hintergrunddienst (`monerod`) allzeit betrieben werden und damit dauerhaft Blöcke senden/empfangen kann, während das Wallet lediglich bei Bedarf - wenn es Zugriff auf die gesamte Blockchain benötigt - zu diesem verbunden werden kann. [Monerujo](https://www.monerujo.io/) sollte auch via [Orbot](https://guardianproject.info/apps/org.torproject.android/) laufen. Da Tors versteckte Dienste Verschlüsselung und Authentifizierung bieten, kannst du dir sicher sein, dass deine RPC-Anmeldedaten nicht ins "Clear(net)" gesendet werden. Tor behebt außerdem Probleme (etwa im Zusammenhang mit Portweiterleitungen oder IP-Adressenwechseln), die häufig bei Privatservern auftreten - es funktioniert einfach. Diese Einrichtung verschleiert zudem den Umstand, dass du zu einem Monero-Remote-Node verbindest. Getestet wurde die Verbindung eines Mac-Laptops zu einem Linux-Remote-Node (Ubuntu 18.04.2) mit der Monero-Version `v0.15.0.1`.
```

Comment:
```suggestion
Unten findest du eine beispielhafte Konfiguration, mit welcher es dir möglich ist, einen Monero-Hintergrunddienst (etwa auf einem Heim-Server oder VPS) zu betreiben, zu dem du dich von einem anderen Rechner, auf welchem dein Wallet läuft, verbinden kannst. Über das Anonymitätsnetzwerk Tor werden die für dein Wallet benötigten Transaktionsinformationen wiederhergestellt. Das Gute an dieser Herangehensweise ist, dass der Hintergrunddienst (`monerod`) allzeit betrieben werden und damit dauerhaft Blöcke senden/empfangen kann, während das Wallet lediglich bei Bedarf - wenn es Zugriff auf die gesamte Blockchain benötigt - zu diesem verbunden werden kann. [Monerujo](https://www.monerujo.io/) sollte auch via [Orbot](https://guardianproject.info/apps/org.torproject.android/) laufen. Da Tors versteckte Dienste Verschlüsselung und Authentifizierung bieten, kannst du dir sicher sein, dass deine RPC-Anmeldedaten nicht ins "Clear(net)" gesendet werden. Tor behebt außerdem Probleme (etwa im Zusammenhang mit Portweiterleitungen oder IP-Adressenwechseln), die häufig bei Privatservern auftreten - es funktioniert einfach. Diese Einrichtung verschleiert zudem den Umstand, dass du dich zu einem Monero-Remote-Node verbindest. Getestet wurde die Verbindung eines Mac-Laptops zu einem Linux-Remote-Node (Ubuntu 18.04.2) mit der Monero-Version `v0.15.0.1`.
```

Diffs don't seem to work here for some reason..

Changed "Privatserver" to "Heim-Server" because VPS also means Virtual "Private Server". "Heim-Server" is a bit unusual too, but I guess it's understandable. Also you can buy "Heim-Server" stuff on Amazon, so I guess it's ok.

Also changed "zu dem du **dich** von einem anderen Rechner verbinden kannst" and "den Umstand, dass du **dich** zu einem Monero-Remote-Node verbindest". I think we were talking about this one before in another PR - here some examples:

![image](https://user-images.githubusercontent.com/6381927/102939110-e8298c00-44ad-11eb-96fa-8b08490401ca.png)